dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -227,33 +227,37 @@ const HorizontalDivider = () => (
 	/>
 );
 
-const podcastImageStyles = (imageSize: MediaSizeType) => {
-	switch (imageSize) {
```

Comment:
ImageSize in ScrollableMedium is `medium` and in a half-width card in FlexibleGeneral is `small`. Images are bigger in the latter.